src/scripts/modules/list.js

```diff
@@ -53,8 +51,28 @@ function renderDestination(destination) {
   const img = document.createElement("img");
   img.style.objectFit = "cover"; // TODO - Prefer adding a CSS class instead.
   img.src = destination.photo;
+  //create checkbox
+  const checkbox = document.createElement("input");
+  checkbox.type = "checkbox";
+  checkbox.classList.add("check");
+
+  // move checked items to 'visited' list
+
+  checkbox.addEventListener("click", () => {
+    toggleButtonVisited(destination);
+    checkbox.parentElement.classList.add("visited");
+    checkbox.parentElement.classList.remove("bucketlist");
```

Comment:
This DOM modification is not needed. After the `toggleButtonVisited()`, the render() gets called again with the list updated (the destination visited will now be toggled), which means the new class will be re-added correctly

Reply:
Thanks Sandrina! I neglected writing things down this time but I didn't forget, I actually had an idea to eventually make the readme file kind of like an article summing everything up. Of course I'll fill in the PR template too

Reply:
In the next PRs, you can update the descriptions as you go, that way you won't forget stuff at the end ;)